src/main/java/com/qa/hubspot/pages/ProfileDemoPage.java

```diff
@@ -0,0 +1,13 @@
+package com.qa.hubspot.pages;
+
+import com.qa.hubspot.base.BasePage;
+
+public class ProfileDemoPage extends BasePage {
```

Comment:
Come on man..where is your by locator ?
Where is your page actions?
Stop writing such non sense codes.